### ISSUE TITLE
Please add Tic Tac Toe please in fun category

### ISSUE THREAD

**Issue by ATS527** [PrajjwalDatir/Kaoi#11]

**Comment by PrajjwalDatir** [comment]
Ok I will try.
Though in Low priority for now, as I am busy with other things.

I saw you are a flutter dev as well as a Typescript developer.
If you are interested in Developing this Command.
Let me know!! 
I can help you with the Baileys and brainstorming!

**Comment by ATS527** [comment]
yeah bro...i used typescript to solve dsa problems..did n't yet worked with node...But i will sure try reading documentations...

**Comment by PrajjwalDatir** [comment]
@ATS527 Started on this?

**Comment by ATS527** [comment]
@PrajjwalDatir no bro....i dont know how to do it like how the chess is implemented...console level was ok...
